src/models/common/decorators/graphQL/classGraphql.decorator.ts

```diff
@@ -19,22 +17,22 @@ export interface IGraphQLClassMapping {
 export function graphqlClass(args?: IGraphQLClass): ClassDecorator {
   // eslint-disable-next-line @typescript-eslint/ban-types
   return <TFunction extends Function>(classCtr: TFunction): TFunction => {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-explicit-any
     const classInstance = new (classCtr as any)();
     const classData: IGraphQLClassMapping = {
       fields: getGraphQLMappings(classInstance),
       name: args?.alias ?? classCtr.name,
     };
 
-    const classDataList = getGraphQLClassMapping() ?? [];
+    const classDataList = getGraphQLClassMapping();
     classDataList.push(classData);
-    Reflect.defineMetadata(graphQLMetadataKey, classDataList, target);
+    target.concat(classDataList);
```

Comment:
concat doesn't update the target in js but returns new array

Reply:
Done, working directly with target